Hackney.Shared.Asset.Tests/Factories/EntityFactoryTest.cs

```diff
@@ -107,5 +107,161 @@ public void CanMapAnAssetTenureToAnAssetTenureDb(bool nullEndDate)
+            domainAC.WindowType.Should().Be(databaseAC.WindowType);
+            domainAC.YearConstructed.Should().Be(databaseAC.YearConstructed);
+            domainAC.AssetPropertyFolderLink.Should().Be(databaseAC.AssetPropertyFolderLink);
+            domainAC.EpcExpiryDate.Should().Be(databaseAC.EpcExpiryDate);
+            domainAC.FireSafetyCertificateExpiryDate.Should().Be(databaseAC.FireSafetyCertificateExpiryDate);
+            domainAC.GasSafetyCertificateExpiryDate.Should().Be(databaseAC.GasSafetyCertificateExpiryDate);
+            domainAC.ElecCertificateExpiryDate.Should().Be(databaseAC.ElecCertificateExpiryDate);
+            domainAC.HasStairs.Should().Be(databaseAC.HasStairs);
+            domainAC.NumberOfStairs.Should().Be(databaseAC.NumberOfStairs);
+            domainAC.HasRampAccess.Should().Be(databaseAC.HasRampAccess);
+            domainAC.HasCommunalAreas.Should().Be(databaseAC.HasCommunalAreas);
+            domainAC.HasPrivateBathroom.Should().Be(databaseAC.HasPrivateBathroom);
+            domainAC.NumberOfBathrooms.Should().Be(databaseAC.NumberOfBathrooms);
+            domainAC.BathroomFloor.Should().Be(databaseAC.BathroomFloor);
+            domainAC.HasPrivateKitchen.Should().Be(databaseAC.HasPrivateKitchen);
+            domainAC.NumberOfKitchens.Should().Be(databaseAC.NumberOfKitchens);
+            domainAC.Kitchenfloor.Should().Be(databaseAC.Kitchenfloor);
+            domainAC.AlertSystemExpiryDate.Should().Be(databaseAC.AlertSystemExpiryDate);
+            domainAC.EpcScore.Should().Be(databaseAC.EpcScore);
+            domainAC.NumberOfFloors.Should().Be(databaseAC.NumberOfFloors);
+            domainAC.Heating.Should().Be(databaseAC.Heating);
+            domainAC.PropertyFactor.Should().Be(databaseAC.PropertyFactor);
+            domainAC.ArchitecturalType.Should().Be(databaseAC.ArchitecturalType);
+            domainAC.AccessibilityComments.Should().Be(databaseAC.AccessibilityComments);
+            domainAC.NumberOfBedSpaces.Should().Be(databaseAC.NumberOfBedSpaces);
+            domainAC.NumberOfCots.Should().Be(databaseAC.NumberOfCots);
+            domainAC.SleepingArrangementNotes.Should().Be(databaseAC.SleepingArrangementNotes);
+            domainAC.NumberOfShowers.Should().Be(databaseAC.NumberOfShowers);
+            domainAC.KitchenNotes.Should().Be(databaseAC.KitchenNotes);
+            domainAC.IsStepFree.Should().Be(databaseAC.IsStepFree);
+            domainAC.BathroomNotes.Should().Be(databaseAC.BathroomNotes);
+            domainAC.LivingRoomNotes.Should().Be(databaseAC.LivingRoomNotes);
+        }
+
+        [Fact]
+        public void AssetCharacteristicsMapsFromDomainToDatabase()
+        {
+            // arrange
+            var domainAC = _fixture.Create<AssetCharacteristics>();
+
+            // act
+            var databaseAC = domainAC.ToDatabase();
+
+            // assert
+            databaseAC.NumberOfBedrooms.Should().Be(domainAC.NumberOfBedrooms);
```

Comment:
Would it not be easier for something like:
`databaseAC.Should().BeEquivalentTo(domainAC)`

Reply:
It's because these 2 models are from 2 separate layers & should not be equivalent in general case.